polyfill/lib/now.mjs

```diff
@@ -0,0 +1,31 @@
+import { ES } from './ecmascript.mjs';
+const Absolute = ES.GetIntrinsic('%Temporal.Absolute%');
+
+export const now = {
+    absolute,
+    dateTime,
+    date,
+    time,
+    timeZone
+};
+
+function absolute() {
+    const ns = ES.SystemUTCEpochNanoSeconds();
+    return new Absolute(ns);
+}
+function dateTime(zone = timeZone()) {
+    zone = ES.ToTimeZone(zone);
+    const abs = absolute();
+    let dateTime = zone.getDateTimeFor(abs);
+    return dateTime;
+}
+function date(zone) {
+    return dateTime(zone).getDate();
```

Comment:
This doesn't seem to be robust against author code messing with prototypes; I didn't quite follow if that was a goal at this point.

Reply:
Should probably discuss. Though I think it's fine for now.